### PR TITLE
Updated logo and homepage charts for BRAIN

### PR DESCRIPTION
**Pull request by abgeorge7**
Description about what this pull request does.

Changes the logo and homepage charts to meeting BRAIN needs.

### Improvements
- Updated logo
- Relevant homepage stats

### CODE REVIEW

File: data/config/bhc.json [hunk 1]
```diff
@@ -70,7 +70,6 @@
       ]
     },
     "navigation": {
-      "title": "BRAIN Commons",
```
Comment (ZakirG) [inline data/config/bhc.json]: why this change?

Comment (abgeorge7) [reply data/config/bhc.json]: Because the logo has the title in it so just going to use that as the nav title.
Similar to NIAID!
